packages/base/src/utils/setMetaTag.ts

```diff
@@ -0,0 +1,39 @@
+/// <reference lib="dom" />
+
+function kebabCase(value: string): string {
+  return value
+    .replace(/([ _A-Z])/gu, '-$1')
+    .replace(/[A-Z]/gu, c => c.toLowerCase())
+    .replace(/([-]+)/gu, '-');
```

Comment:
Sorry, that didn't mean to be direct suggestion 😄 

If you want one which actually works well, consider:

```suggestion
    .replace(/([a-z0-9]|(?=[A-Z]))([A-Z])/gu, '$1-$2')
    .toLowerCase()
    .replace(/[^a-z0-9]+/gu, '-')
    .replace(/^-+|-+$/gu, '');
```